x-pack/test/cases_api_integration/common/lib/connectors.ts

```diff
@@ -0,0 +1,324 @@
+/*
```

Comment:
Moved all the connector related utility functions for the integrations tests to this file